arkouda/categorical.py

```diff
@@ -1356,3 +1359,51 @@ def parse_hdf_categoricals(
                 result_categoricals[base_name] = Categorical.from_codes(**cat_parts)
 
         return removal_names, result_categoricals
+
+    def send_array(self, hostname: str, port: int_scalars):
+        """
+        Sends a Categorical object to a different Arkouda server
+
+        Parameters
+        ----------
+        hostname : str
+            The hostname where the Arkouda server intended to
+            receive the Categorical is running.
+        port : int_scalars
+            The port to send the array over. This needs to be an
+            open port (i.e., not one that the Arkouda server is
+            running on). This will open up `numLocales` ports,
+            each of which in succession, so will use ports of the
+            range {port..(port+numLocales)} (e.g., running an
+            Arkouda server of 4 nodes, port 1234 is passed as
+            `port`, Arkouda will use ports 1234, 1235, 1236,
+            and 1237 to send the array data).
```

Comment:
i don't know much about networking but i wanted to throw this out there. Idk if it's a reasonable assumption that a block of num locales ports would be free. I know that num_locales can get pretty big in real use cases, so I just wanted to check that this was something ppl who know things about networks had thought about lol